draft-link-v6ops-464xlaton.xml

```diff
@@ -436,6 +445,11 @@ If the attacker intercepts traffic for the NAT64 prefix (e.g. by providing the v
 </ul>
 <t>
 Using the PREF64 RA option to detect PLAT presence and the NAT64 prefix is less prone to such attacks, as the attacker needs to be on-link and be able to bypass layer-2 security features such as RA Guard.
+</t>
+<t>
+In networks lacking explicit IPv6 deployment, administrators may inadvertently expose link-local IPv6 connectivity when endpoints have IPv6 enabled.
+This unintended exposure can facilitate PLAT presence signal falsification, enabling malicious actors to conduct Man-in-the-Middle (MitM) or Denial-of-Service (DoS) attacks.
+To mitigate this risk, it is recommended to disable CLAT upon detecting native IPv4 connectivity, as outlined in <xref target="disable"/>.
```

Comment:
```suggestion
This document mitigates this risk by requiring endpoints to disable CLAT when the network provides non-link-local IPv4 connectivity, as outlined in <xref target="disable"/>.
```

I get that this is not a normative "recommend" but this is still confusing because RECOMMENDED means SHOULD, but we clearly have disabling CLAT in favor of native IPv4 connectivity as a MUST now. How about this?